app/src/main/java/com/cornellappdev/android/eatery/CampusMenuActivity.java

```diff
@@ -207,6 +226,109 @@ else if (mCafeData instanceof DiningHallModel) {
                         ContextCompat.getColor(getApplicationContext(), R.color.blue));
             }
         }
+
+        setupWaitTimesChart();
+    }
+
+    private void setupAxis(BarChart b) {
+        // Remove bar chart Y axis - left side.
+        YAxis yAxisLeft = b.getAxisLeft();
+        yAxisLeft.setDrawGridLines(false);
+        yAxisLeft.setDrawLabels(false);
+        yAxisLeft.setEnabled(false);
+        yAxisLeft.setAxisMinimum(-0.5f);
+
+        // Remove bar chart Y axis - right side.
+        YAxis yAxisRight = b.getAxisRight();
+        yAxisRight.setDrawGridLines(false);
+        yAxisRight.setDrawLabels(false);
+        yAxisRight.setEnabled(false);
+
+        // Customize bar chart X axis.
+        XAxis xAxis = b.getXAxis();
+        xAxis.setDrawGridLines(false);
+        xAxis.setPosition(XAxis.XAxisPosition.BOTTOM);
+
+        // Set X axis colors and fonts.
+        // "xAxis.set[_]Color(R.color.[_])" will use Android's default colors, rather than our custom defined.
+        xAxis.setAxisLineColor(getResources().getColor(R.color.inactive));
+        xAxis.setTextColor(getResources().getColor(R.color.secondary));
+        xAxis.setTextSize(14f);
+        xAxis.setAxisLineWidth(2f);
+        // Setup X axis labels.
+        xAxis.setAxisMaximum(21);
+        xAxis.setLabelCount(8);
+        xAxis.setTypeface(Typeface.SANS_SERIF);
+        xAxis.setYOffset(8f);
+        ArrayList<String> xAxisLabels = new ArrayList<String>();
+        for(int i = 0; i < 22; i++) {
+            int hourNum = (i + 6) % 12;
+            if (hourNum == 0) {
+                hourNum = 12;
+            }
+            String ap = i <= 5 || i >= 18 ? "a" : "p";
```

Comment:
i'm not a huge fan of the am/pm label on the graphs. is that what the design shows?

Reply:
Yeah, this is the design:

![image](https://user-images.githubusercontent.com/43295248/66583487-6cb21800-eb51-11e9-9088-00a61ab52fbc.png)

Reply:
aight.